webapp/channels/src/packages/mattermost-redux/src/selectors/entities/scheduled_posts.ts

```diff
@@ -56,13 +56,18 @@ export function hasScheduledPostError(state: GlobalState, teamId: string) {
 }
 
 export function showChannelOrThreadScheduledPostIndicator(state: GlobalState, channelOrThreadId: string): ChannelScheduledPostIndicatorData {
-    const channelScheduledPosts = state.entities.scheduledPosts.byChannelOrThreadId[channelOrThreadId] || emptyList;
+    const allChannelScheduledPosts = state.entities.scheduledPosts.byChannelOrThreadId[channelOrThreadId];
+    const eligibleScheduledPosts = allChannelScheduledPosts.filter((scheduledPostId: string) => {
```

Comment:
```suggestion
    const eligibleScheduledPosts = allChannelScheduledPosts?.filter((scheduledPostId: string) => {
```

Reply:
AH so yeah that was the bug. I restored the original `|| emptyLIst` part so we don't need the ?. Just tha that empty list part was there before, thats why, essentially both are similar